nucypher/policy/models.py

```diff
@@ -263,7 +267,7 @@ def enact(self, network_middleware, publish=True) -> dict:
         else:  # ...After *all* the policies are enacted
             # Create Alice's revocation kit
             self.revocation_kit = RevocationKit(self, self.alice.stamp)
-            self.alice.active_policies[bytes(self.public_key).hex()] = self
+            self.alice.active_policies[self.id] = self
```

Comment:
I get that we're setting the new active policies on Alice here, but it reads a bit awkward.  Still massive improvement from `self.alice.active_policies[bytes(self.public_key).hex()] = self`.  I know it's a tiny adjustment, but perhaps moving `self.alice.active_policies[self.id] = self` to an alice method will be more semantic.

Reply:
I can get behind that. You mean add two methods for policy addition and deletion?

Reply:
Yea - may provide a place to handle errors, too.